mlflow/projects/databricks.py

```diff
@@ -77,7 +77,13 @@ def _upload_to_dbfs(self, src_path, dbfs_fuse_uri):
         _logger.info("=== Uploading project to DBFS path %s ===", dbfs_fuse_uri)
         http_endpoint = dbfs_fuse_uri
         with open(src_path, 'rb') as f:
-            self._databricks_api_request(endpoint=http_endpoint, method='POST', data=f)
+            try:
+                self._databricks_api_request(endpoint=http_endpoint, method='POST', data=f)
```

Comment:
could use the unsafe version so we can get the error code directly, but seems about the same

Reply:
yea using the unsafe version seems nice, but if we're gonna check for "File already exists" in the message (which seems reasonable to help avoid silent/confusing failures in this race-condition edge case), it seems fine to go with the current approach.